Octokit/Clients/IPackagesClient.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace Octokit
+{
+    public interface IPackagesClient
+    {
+        Task<IReadOnlyList<Package>> List(string org, PackageType packageType, PackageVisibility? packageVisibility = null);
```

Comment:
Do you think `List` is the right name here? Should we use `ListForOrganization` or something like that, given that there are other list APIs?

Reply:
Yeah it wasn't the best name. I believe I changed it in later commits to "GetAllForOrg" and so on for the various combinations.